common/src/thrift/block_master.thrift

```diff
@@ -15,49 +15,69 @@ struct WorkerInfo {
 
 service BlockMasterService extends common.TachyonService {
 
-  // Tachyon Client API
-
-  /*
+  /**
    * Returns the block information for the given block id.
+   * @param blockId
```

Comment:
so do you just want to leave the description itself, removing all return, param, and throws? Let me know. @jsimsa @calvinjia